gprofiler/profilers/perf.py

```diff
@@ -12,14 +12,36 @@
 
 from gprofiler.log import get_logger_adapter
 from gprofiler.merge import ProcessToStackSampleCounters, merge_global_perfs
-from gprofiler.profiler_base import ProfilerBase
+from gprofiler.profilers.profiler_base import ProfilerBase
+from gprofiler.profilers.registry import ProfilerArgument, register_profiler
 from gprofiler.utils import TEMPORARY_STORAGE_PATH, resource_path, run_process
 
 logger = get_logger_adapter(__name__)
 
 PERF_BUILDID_DIR = os.path.join(TEMPORARY_STORAGE_PATH, "perf-buildids")
+PERF_MODE_ARGUMENT_HELP = (
```

Comment:
What's the use of placing it here? This "const" is meaningful next to the other related arguments (e.g `possible_modes`). Just embed it inside `register_profiler(...)`

Reply:
Moved it because it's a big string, I'll move it back

Reply:
I don't think there's use in moving away big strings. This number ain't a "meaningful" const like e.g `PERF_BUILDID_DIR` one line above it. It's just a string and it's easier to read the string right next to where it's used.

Reply:
Yeah, I agree. I wanted to shorten the huge register line, but in this case it only hurt readablity